src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -106,6 +106,39 @@ class CommonUtils {
     }
 
     /**
+     * Utility to add a performance publisher block, for use in performance tests
+     *
+     * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">Common utils</a>
+     */
+
+    static void addPerformancePublisher(context, String reportPattern="**/results/*.jtl", String unstableResponseTimeThreshold="", int failedThresholdPositive, int failedThresholdNegative, int unstableThresholdPositive, int unstableThresholdNegative) {
```

Comment:
It's just less clear that's all.  Every time you use the function, you'll have to go back and look it up, like what do I pass in again?  It's more of a suggestion though.  Currently, you're passing in 7 parameters, and let's just say you want to add more at a later time.  The function approach is less extendable.